### PR TITLE
New spinning cube with custom behavior

### PR DESCRIPTION
**Pull request by amberroy**

### CODE REVIEW

File: examples/spinning-cube.html [hunk 1]
```diff
@@ -66,5 +56,31 @@
 	}
 }
 
+ChangeColor = function () {//define a custom behavior
```
Comment (brianpeiris) [inline examples/spinning-cube.html]: `function ChangeColor () {` ?

**Comment by amberroy** [comment]
thanks Brian, I changed it to a standard function def.

File: examples/spinning-cube.html [hunk 2]
```diff
@@ -66,5 +56,31 @@
...
+  var lastColor;
+
+  function awake(o) {
+    object3d = o;
+		object3d.addEventListener('cursordown', function() {
+			object3d.userData.syncData.color = Please.make_color()[0];//random cololr
+			var sync = object3d.getBehaviorByType('Object3DSync');//TODO: better way of doing this
+      if (sync) sync.enqueueSend();
+		});
+  }
```
Comment (brianpeiris) [inline examples/spinning-cube.html]: Mixing tabs and spaces.

Comment (amberroy) [inline examples/spinning-cube.html]: Changed to tabs only.

**Pull request merged**